client/app/src/main/java/com/example/criengine/Adapters/RequestsForBookAdapter.java

```diff
@@ -100,20 +94,18 @@ public void onClick(View v) {
             new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    // TODO: Notify the user of the rejection.
-                    userRequests.remove(name);
-                    // TODO: Push changes to database.
+                    dbw.declineRequest(name, book.getBookID());
 
+                    userRequests.remove(name);
```

Comment:
Just checking, you remove the requester here, but then on https://github.com/CMPUT301F20T05/CriEngine/pull/163/files#diff-771beae1af671b59afa5f6fede78f03323b8eefc98303f4a1a0c61d291965dc5R83
then you don't clear the list there? 
Have you already accounted for that?

Reply:
We dont wanna remove the entire list. Each rejection removes one, leaving only the request of the accepted user. If we removed them from the list the book wouldn't know who wac accepted to borrow it

Reply:
Ah I see, so you intentionally remove users in the second one so that it can update the screen on the spot but then when you actually accept someone then it will go to another screen so it doesn't matter